### ISSUE TITLE
Talent Details

### ISSUE THREAD

**Issue by Tarkisflux** [SqueeG/awesomeTome#20]
Everyone seems on-board with the idea of skill talents as extensions of skills and a place for non-skills to live. So let's sort a couple of other issues with them, before the work of transcribing them gets underway.

1) Should we remove int bonus to skills and replace it with int bonus to traits?
2) Should prereqs be skill ranks only, or skill ranks + maybe traits, or have some other limitation? Some of the epic stuff we're going to move in might be well served by having previous traits as prereqs on top of ranks, but it's an extra cost to ability acquisition.
3) Except for craft, the partial benefit lines seem to be written for proficiencies. Do we want to remove partial benefit entirely, and just leave full benefits (with craft just having advanced and regular options to buy)?
4) Name. Talents is probably fine, but other options include 'skill tricks' or 'skill exploits', and I'm sure there are others we could grab.

My preferences are below:
1) Yes, yes we should.
2) Skill ranks + traits + maybe BAB in some really rare cases. I think it promotes slightly more organic growth of traits, and the increased cost just gives a place for int bonuses to go.
3) Remove partial benefits; advanced craft just requires regular craft.
4) Skill exploits is my preference, but not by much.

**Comment by ExplosiveRunes** [comment]
1. Yes we should
2. There's no reason to explicitly limit the requirements traits can have. We may decide for the purpose of the TSRD to limit the way in which we use requirements on them for standardization, but someone may decide to make new traits that interact with classes or feats to a greater degree, this should be allowed.
3. It was mostly for proficiencies. But it is noteworthy that without the Partial Benefit you might run into a situation where a character invests a point into a trait/talent and gets nothing for it that level. If you want to change craft, I'd say have it require a a related profession to mastercraft. That way you don't need a second tier of talents.
4. I'm still partial to talents, as I obviously would be. Traits is a word already used by races, and exploits seemd to refer to things you get for free through in-game action (not something you spend skill points on).  I'd say that if we want keep proficiencies as exploits, we need to more explicitly codify exploits, and use them for something other than weapon proficiencies.

**Comment by Tarkisflux** [comment]
Ugh. It was talents wasn't it. Stupid brain.

And it has been edited to reflect the actual name, and not the thing stuck in my brain for no good reason.

**Comment by SqueeG** [comment]
So wait... there are skill, traits and talents?
I thought traits and talents were the same thing.

**Comment by ExplosiveRunes** [comment]
No, just a discussion of naming. Same thing.

**Comment by Tarkisflux** [comment]
@SqueeG 
Yeah, for some reason I got "skill traits" stuck in my head when I wrote that, and that's not what they were called and I was dumb.

@ExplosiveRunes 
I don't think we really need to worry about expanding the exploit system, because I don't see a lot of value in weapon proficiencies. We could expand to armor as well maybe if it was important though...

Guess we should put up another discussion point:
5) Should we keep weapon proficiencies as exploits (and maybe do something similar for armor and shields), or adopt @ExplosiveRunes idea and merge proficiencies into the talent system?

My answer:
5) I'm not sure. As a talent it's a decent low level place for point investment before higher level stuff becomes available, and we could probably use a couple more of those. But it also means that proficiencies are approximately as valuable or time intensive as a profession or language or whatever else, and I'm not sure that's the case or even desirable. The cost also prevents thugs from learning new weapons on the fly, restricting them to only at level up instead, and that seems a step back from exploits. Not sure if there's anything in the new weapon system to mitigate that though.

**Comment by SqueeG** [comment]
Where ever possible don't make a step backward by restricting stuff the Tomes have already opened up. Exploits are a pretty simple solution. Fleshing it out better is a good idea, though.

Unless that was already done and I just never read it.

**Comment by ExplosiveRunes** [comment]
@Tarkisflux Somehow, I thought I'd responded to this. Thugs still don't get to wield weapons on the fly, unless they have martial weapon proficiency, and have a week of time to train. To quote the tomes:
- "If you don't have Martial Weapon Prociency and you want to use a new weapon, that's touchier. But if you
  have a weapon for an entire level, you should just gain prociency in it when you gain your next level whatever level
  you happen to select."

That's on a per-level basis too. And lets be honest, if the DM wanted thugs to have specific weapons, they'd just come onto the scene with that exploit/that talent anyway. If you want _your_ thugs to be having certain weapons, you're probably telling them "You fucks better choose the talent I tell you to, or I quit paying you and replace you with minotaurs" (Or however you would actually say that in character). Which seems fine, your thugs level up about as often as you do with leadership. I'd also like to point out that if you invest bonus skill points into, say, martial weapon proficiency, and later _gain_ martial weapon proficiency from another source (say, from taking a level of something else) you'd get those points back and could immediately stick them into some other talent. I'd imagined it as an easy way to even out a character concept at low levels if you always intended to go down a certain route. Of course, someone seriously interested in character optimization could pick up all the necessary proficiencies for eldritch knight as a wizard to avoid losing a caster level. Is 10 skill points worth a caster level? Yeah, probably.

If we do keep exploits, I'd like to see it become a more interesting, but still minor, subsystem for learning minor things on the fly. Weapons is a good example, as is shields and armor (of no higher a category than the one above what you currently have).  But I'd also like to see teaching the local militia that you can, in fact, score critical hits on zombies (but teaching them how to score critical hits on ghouls would take another whole week).
